downstream/modules/eda/proc-eda-build-a-custom-decision-environment.adoc

```diff
@@ -16,13 +16,16 @@ You can create a custom decision environment for {EDAName} that provides a custo
 
 .Procedure
 
-* Add the `de-supported` decision environment. 
-This image is built from a base image provided by Red Hat called `de-minimal` at link:https://catalog.redhat.com/software/containers/ansible-automation-platform-24/de-supported-rhel9/6449642e3993031ccdf4cf19[Ansible Automation Platform supported decision environment].
+* Use `de-minimal` as the base image with {Builder} to build your custom decision environments. 
```

Comment:
It is confusing to user the name of base image in this document. Here and the examples use `de-minimal-rhel9`, while in the IMPORTANT section `de-supported-rhel9` is used. Which base image should be used `de-minimal` or `de-supported`?

Reply:
Hi @bzwei . Thank you for your comments. By the time you were posting your message, we had move forward with our publishing process. I believe you ask a valid question and I attempted to make that clear in the content below letting the user know that they would use the `de-minimal` image that is provided in the registry.redhat.io by the `de-supported` images. I was confused initially, but thought tweaking the sentence below made it clearer that what the user should use is `de-minimal`. We can discuss this further, if necessary, in the "parking lot" portion of our stand-up, Wednesday, Nov. 13th.